src/client/pages/OfferNew/Checkout/InsuranceSummaryDetails.tsx

```diff
@@ -200,6 +203,29 @@ const getQuoteDetails = (
   return detailsGroups
 }
 
+export const getAddress = (quoteDetails: QuoteDetails) => {
+  if (!quoteDetailsHasAddress(quoteDetails)) {
+    return null
+  }
```

Comment:
If we are really sure we wouldn't call this function with a quote lacking address:

```suggestion
  if (!quoteDetailsHasAddress(quoteDetails)) {
    throw new Error("Quote details need to include address fields")
  }
```